src/main/java/dk/cs/aau/huppaal/controllers/HUPPAALController.java

```diff
@@ -764,69 +731,50 @@ protected void interpolate(final double frac) {
                 tabPaneContainer.setMaxHeight(((height - 35) * (1 - frac)) + 35);
             }
         };
+    }
 
-        if (tabPaneContainer.getMaxHeight() > 35) {
+    public void collapseMessagesIfNotCollapsed() {
+        var collapse = collapseTransition();
+        if (tabPaneContainer.getMaxHeight() > 35)
             collapse.play();
-        }
     }
 
     @FXML
     public void collapseMessagesClicked() {
-        final Transition collapse = new Transition() {
-            double height = tabPaneContainer.getMaxHeight();
-
-            {
-                setInterpolator(Interpolator.SPLINE(0.645, 0.045, 0.355, 1));
-                setCycleDuration(Duration.millis(200));
-            }
-
-            @Override
-            protected void interpolate(final double frac) {
-                tabPaneContainer.setMaxHeight(((height - 35) * (1 - frac)) + 35);
-            }
-        };
-
-        if (tabPaneContainer.getMaxHeight() > 35) {
+        var collapse = collapseTransition();
+        if (tabPaneContainer.getMaxHeight() > 35)
             collapse.play();
-        } else {
+        else
             expandMessagesContainer.play();
-        }
     }
 
     @FXML
     private void generateUppaalModelClicked() {
-        final Component mainComponent = HUPPAAL.getProject().getMainComponent();
-
+        var mainComponent = HUPPAAL.getProject().getMainComponent();
         if (mainComponent == null) {
-            HUPPAAL.showToast("Cannot generate UPPAAL file without a main component");
-            return; // We cannot generate a UPPAAL file without a main component
+            HUPPAAL.showToast("no main component");
```

Comment:
Capitalize 'N' in "no"

Reply:
Fixed in 4a4eaa25a8ec00e3b3fa4439a3e7078ea4d188fb